pages/api/utils/get_more_data.tsx

```diff
@@ -0,0 +1,22 @@
+import hasNoUserAccess from '../../../lib/utils'
```

Comment:
Let’s find a better name for this file. get_more_data is too generic

Reply:
Also is snake_case the naming convention for js? Can you please do some research on it?

Reply:
"load_more_bookings" is ok?

Reply:
"File names must be all lowercase and may include underscores (_) or dashes (-), but no additional punctuation." This is what I found

Reply:
buuun, good job. Acum am vazut ca asa e si in coding style-ul de la google

Reply:
Let’s call it get_paginated_data